src/Evolution/Systems/CurvedScalarWave/Tags.hpp

```diff
@@ -40,4 +41,23 @@ struct ConstraintGamma2 : db::SimpleTag {
   using type = Scalar<DataVector>;
   static std::string name() noexcept { return "ConstraintGamma2"; }
 };
+
+/*!
+ * \brief Tags corresponding to various constraints of the scalar
+ * wave system, and their diagnostically useful combinations.
+ * \details For details on how these are defined and computed, see
+ * `OneIndexConstraintCompute`, `TwoIndexConstraintCompute`
```

Comment:
point to relevant hpp file where these are defined?

Reply:
The " `OneIndexConstraintCompute`, `TwoIndexConstraintCompute`" will show up as clickable links pointing to pages documenting these 2 compute items and the header files they are defined in.. I think adding hpp file names here might be overkill, see a similar example [here](https://spectre-code.org/structgeneralizedharmonic_1_1tags_1_1fourindexconstraint). If you still feel strongly about this, I can add that info.